kubejs/assets/ftbquests/lang/pl_pl.json

```diff
@@ -0,0 +1,130 @@
+    "chapter.1.quest.17.subtitle": "Mniam ryż",
+    "chapter.1.quest.17.title": "Uzyskaj nowe materiały",
+    "chapter.1.quest.18.description.1": "Wszyscy moi koledzy nienawidzą stylu na początku gry, idźmy szukać podróżnego stroju dla prawdziwego stylu.\n\nZbuduj kompletny zestaw podróżnego ekwipunku i dam ci nagrodę. Będziesz musiał szukać w REI i książkach Tinkers' samemu. To sprawiedliwe, prawda?",
+    "chapter.1.quest.19.description.1": "\n\nLizardLeliel - Gra ważną rolę w postępach zadań w rozdziałach 1-3 i jest odpowiedzialny za dużą część przepisów i automatyzacji, które możesz znaleźć w pakiecie!\n\n\nEthanicuss - Rozwój struktury datapack (astralrails), wkład muzyczny, pomoc w zadaniach, wiele różnych wpisów w trakcie rozwoju (BARDZO).\n\n\nIris - Niesamowity wkład w muzykę pakietu, w tym oficjalną ścieżkę dźwiękową Create: Astral! Odkryj ją na Spotify przed jej implementacją!\n\n\nChoconeosome - Budowa większości budynków, które zobaczysz w AstralRails!",
+    "chapter.1.quest.19.subtitle": "Laky docenia WSZYSTKICH",
+    "chapter.1.quest.19.task.1.title": "Rzeczy do zapamiętania!",
+    "chapter.1.quest.19.title": "Specjalne podziękowania",
+    "chapter.1.quest.2.description.1": "Konstruktor części jest miejscem, w którym bierzesz wzór, bierzesz materiały bazowe i tworzysz indywidualne komponenty narzędzia.",
+    "chapter.1.quest.2.subtitle": "Zbuduj części swojego narzędzia z tym.",
+    "chapter.1.quest.20.title": "Zbierz! 64 bloki melona",
+    "chapter.1.quest.21.title": "Zbierz!! 64 kaktusy",
+    "chapter.1.quest.22.title": "Zbierz!! 64 dynie",
+    "chapter.1.quest.23.title": "Zbierz!! 64 skrzynki marchewkowe",
+    "chapter.1.quest.24.title": "Zbierz!! 64 skrzynki ziemniaczane",
+    "chapter.1.quest.25.title": "Zbierz!! 64 skrzynki kukurydziane",
+    "chapter.1.quest.26.title": "Zbierz!! 64 worki ryżowe",
+    "chapter.1.quest.27.title": "Zbierz!! 64 skrzynki cebulowe",
+    "chapter.1.quest.28.title": "Zbierz!! 64 skrzynki pomidorowe",
+    "chapter.1.quest.29.title": "Zbierz!! 64 skrzynki kapuśniaczane",
+    "chapter.1.quest.3.description.1": "Stacja Tinker jest miejscem, w którym montujesz swoje narzędzia Tinkers', które będą potrzebne do uzyskania kamienia, miedzi i żelaza. Narzędzia drewniane i kamienne są pierwszym celem, ponieważ receptury vanilla tych narzędzi już nie istnieją.\n\n\nMożesz również naprawiać narzędzia Tinkers' w tej stacji, umieszczając swoje narzędzie w prawej szczelinie i materiały w dowolnej innej szczelinie. To jest super wygodne! Możesz również zainwestować w zestawy naprawcze, aby naprawiać narzędzia Tinkers' bezpośrednio z twojego ekwipunku!",
+    "chapter.1.quest.3.subtitle": "Zmontuj swoje narzędzia z tym.",
+    "chapter.1.quest.30.description.1": "(Och, to było warte tego.)",
+    "chapter.1.quest.30.subtitle": "Czy to było warte?",
+    "chapter.1.quest.30.title": "Bardzo ładny przypraw",
```

Comment:
What. the. heck. is this translation.

"przyprawa" is feminine, and this refers to mutliple spices, so this should probably be "Bardzo ładne przyprawy".

Reply:
Okay this is getting embarrassing XDD how did i miss it...